.yarnclean

```diff
@@ -0,0 +1,44 @@
+# Exclude some needed stuff
```

Comment:
Do we need this file with all it's unrelated entries?

Reply:
This file helps to make the `node_modules` folder a bit smaller after an install